packages/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -329,10 +324,22 @@ $fullscreen-modal-padding: 64px;
 
 .header-content {
   display: inline-block;
-  margin-top: var(--spacing-xxs, spacing(xxs))
+  margin-top: var(--spacing-xxs, spacing(xxs));
+  padding-left: 6px;
```

Comment:
or `pl-6` on the header itself, but entirely up to you

Reply:
`pl-6` would actually equate to `32px` 😃 

![image](https://user-images.githubusercontent.com/67973710/176690832-64b5350d-24bb-4eab-aca0-4b5d86be4601.png)